### PR TITLE
Withdraw work item

### PR DESCRIPTION
**Pull request by OR13**
Per the discussions and resolutions at the Miami Face to Face, I am raising this PR to complete the changes necessary to withdraw this work item.

@iherman please let me know what additional changes I need to make.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/vc-jws-2020/pull/33.html" title="Last updated on May 5, 2023, 3:58 PM UTC (8c0f0ee)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/vc-jws-2020/33/624333d...8c0f0ee.html" title="Last updated on May 5, 2023, 3:58 PM UTC (8c0f0ee)">Diff</a>

### CODE REVIEW

**Comment by brentzundel** [comment]
@OR13 Thank you for starting this process.
The official guidance is here: https://www.w3.org/2021/Process-20211102/#abandon-draft

**Comment by OR13** [comment]
@brentzundel @iherman it does not appear that there is a "status" for "discontinued" / "abandoned".

https://respec.org/docs/#specStatus

It seems the guidance is to downgrade to an en ED...

https://www.w3.org/2021/Process-20211102/#discontinuedREC

File: index.html
Comment (iherman) [inline index.html]: There is no such thing as a separate spec status for discontinued REC. Just leave this field as "ED", the Echidna process will turn this into a Working draft, and it will stay that way. And that is perfectly fine.

Comment (iherman) [inline index.html]:
```suggestion
        specStatus: "ED", 
```

**Review by iherman — CHANGES_REQUESTED** [review]

File: index.html
Comment (iherman) [inline index.html]: This section should be copied and, possibly, extended in the status section. Why has it been discontinued, why is it advised to look at vc-jwt?

Technically, a paragraph or two should be added stod section and respec will add the boilerplate.

However. The text refers to https://w3c.github.io/vc-jwt/, which is the editor's draft, and that is wrong. It should be the URL of the latest, official version, ie, https://www.w3.org/TR/vc-jwt/. Except that this does not exist yet (how come?). Ie, I believe the right sequence of actions is:

1. Publish, formally, vc-jwt as a FPWD, because that is what should be the official target
2. _Once that has happened,_ change the URL in this remark to point to vc-jwt, and then it can be published as an official Working Draft with the withdrawal notice.

**Review by iherman — CHANGES_REQUESTED** [review]

File: index.html
Comment (iherman) [reply index.html]: @brentzundel @Sakurann

Comment (OR13) [reply index.html]: @iherman yes, agree with your suggestions, I can't ref vc-jwt until after FPWD technically, thats why it is pointing to github now.

I will make the changes assuming vc-jwt goes to FPWD and then we can hold this PR till then.

File: index.html
Comment (OR13) [reply index.html]: merged your suggestion, resolving.

File: index.html
Comment (OR13) [inline index.html]:
```suggestion
        The working group recommends using <a href="https://www.w3.org/TR/vc-jwt">vc-jwt</a> 
```

Comment (OR13) [reply index.html]: This is the change I will wait to merge until after vc-jwt is FPWD.

**Comment by brentzundel** [comment]
What are the next steps that need to happen here? @iherman @OR13

**Comment by iherman** [comment]
> What are the next steps that need to happen here? @iherman @OR13

Publish a new draft (ie, merge the repo) with all the changes that we have agreed upon.

Once it is published, it is probably worthwhile to turn the repository into an ["archive"](https://docs.github.com/en/repositories/archiving-a-github-repository/archiving-repositories). The github page suggests (but does not require) to close all issues and PR; we may want to look at this for this repo, but (without looking) there may be issues that we prefer to leave open.

As a precautionary measure it may be a good idea to disable the echidna action _before_ archival. This would prevent some unwanted publication in case in several years someone decides to reopen the repo...

**Comment by OR13** [comment]
@iherman  I noticed the resolutions here are linking incorrectly:

https://www.w3.org/2017/vc/WG/Meetings/Minutes/resolutions#resolutions-taken-at-regular-working-group-calls
Screen Shot 2023-05-05 at 10 58 49 AM

**Comment by OR13** [comment]
@iherman after this PR is merged,  I will remove echidna, and I suggest the repo be made read only.

**Comment by brentzundel** [comment]
@OR13 not sure what needs to happen to get this merged.

**Comment by OR13** [comment]
@iherman please re-review.

File: index.html
Comment (iherman) [inline index.html:102]:
```suggestion
        <a href="https://www.w3.org/2023/Process-20230612/#discontinuedREC">discontinued</a>. 
```

(The new process has just been published... ☹️)

**Review by iherman — APPROVED** [review]

**Comment by iherman** [comment]
> @iherman please re-review.

Just did, sorry for the delay, I had/have some internet issues.

Apart from the minor comment above, it is a go. Ping me when done, I think the repository should be "archived", in github parlance.

**Review by Sakurann — APPROVED** [review]
please merge and withdraw/archive

**Comment by iherman** [comment]
@OR13 ping?

**Comment by OR13** [comment]
@iherman let me know if you need anything else from me

**Comment by iherman** [comment]
Thank @orie. Do you want to make some last minute adjustment on the repository site? If not, I will archive it.

**Comment by OR13** [comment]
archive away!

**Comment by iherman** [comment]
Consider it done!

**Pull request merged**
